openmc/deplete/nuclide.py

```diff
@@ -212,6 +215,14 @@ def from_xml(cls, element, fission_q=None):
 
         fpy_elem = element.find('neutron_fission_yields')
         if fpy_elem is not None:
+            # Check for use of FPY from other nuclide
+            parent = fpy_elem.get('parent')
+            if parent is not None:
+                fpy_elem = root.find(
+                    './/nuclide[@name="{}"]/neutron_fission_yields'.format(parent)
+                )
```

Comment:
Two potential errors that could arise here.
1. `parent` is found but `root is None`  :arrow_right: `None.find(...)`
2. `parent` does not exist in the file :arrow_right: `FYD.from_xml_element(None)` :arrow_right: `None.iter(...)`

Since this is largely an internal method, the likelihood might be low. I think some `assert` statements would be okay, just to provide errors that are more insightful

Reply:
Good points -- will add some code to handle those

Reply:
Just added a commit to address these